components/patient-data-sharing/receiver-api/src/main/resources/ApplicationResources_es.properties

```diff
@@ -1,8 +1,10 @@
 # The title of the administrative section
 admin.receivepatientconfig=Recibir la configuraci\u00f3n del paciente
+phenotips.patientSharing.receive.configuration.title=Recibir la configuraci\u00f3n
 phenotips.patientSharing.receive.configuration.whitelist.title=Servidores de base autorizados
 phenotips.patientSharing.receive.configuration.submit=Guardar
 
+PhenoTips.ReceivePatientSettings_AllowPushesFromNonListedServers=Permitir que los empujes de los servidores no se enumeran a continuaci\u00f3n
 PhenoTips.ReceivePatientSettings_AllowNoConsentsFromOldClients=Aceptar datos de los clientes antiguos que no pueden empujar informaci\u00f3n del consentimiento
 
 PhenoTips.ReceivePatientServer_description=Descripci\u00f3n
```

Comment:
These aren't good, they have a different meaning. I left them out explicitly from the previous pull request, since a native Spanish speaker should translate them.